### ISSUE TITLE
Visually differentiate registryGroups

### ISSUE THREAD

**Issue by nichtich** [gbv/cocoda#495]
The badges of registries (L, C, CO, WR...) should get their their color from registryGroups in configuration file (related: #305) to better differentiate mappings and mapping recommendations.

**Comment by nichtich** [comment]
See #500 instead.

**Comment by nichtich** [comment]
Depends on #500 but this issue is more about user interface. User feedback suggest we need a more visual differentiation between "databases" of mappings and "recommendations" for mappings. Maybe a vertical bar or splitting into sub-tables?

**Comment by stefandesu** [comment]
I have a suggestion for this:

```
Mappings: C L W
-----
... table with mappings from databases ...
-----

Recommendations: CR CC CO
-----
... table with mapping recommendations ...
-----
```

If we removed the table headers (yes I know that I spoke against this in another issue 🙈), it wouldn't change much space-wise, and it would be a very clear distinction between database mappings and mapping recommendations. We could also add a button to show/hide either (and maybe hide recommendations by default). 

What do you think? @nichtich

**Comment by stefandesu** [comment]
I adjusted Mapping Navigator as described in my previous comment. A few notes:

- I removed the table headers for all tables, including the Mapping Search.
- Because of the removal of table headers, the Share Search button in Mapping Search had no place anymore. I temporarily removed it, but I will think about a new place for it and add it again.
- Currently, it is not possible to hide database mappings or recommendations. (Will be added after #468 is implemented.)
- I think the labels should be adjusted.
- When testing, please make sure that scrolling in Search and Navigator work properly, especially in different browsers.

I think it was a good idea to separate the tables. After adjusting the labels, it will be a much clearer differentiation between database mappings and mapping recommendations.

**Comment by nichtich** [comment]
Looks good. I did some minor layout changes. Labels are fine.
